packages/storybook/vite.config.js

```diff
@@ -9,7 +9,8 @@ export default defineConfig({
   resolve: {
     alias: {
       '@us-gov-cdc/cdc-react': path.resolve(__dirname, '../cdc-react/src/components/index.ts'),
-      '@us-gov-cdc/cdc-react-icons': path.resolve(__dirname, '../cdc-react-icons/src/components/index.ts')
+      '@us-gov-cdc/cdc-react-icons': path.resolve(__dirname, '../cdc-react-icons/src/components/index.ts'),
+      'uswds-core': path.resolve(__dirname, '../cdc-react/node_modules/@uswds/uswds/packages/uswds-core')
```

Comment:
I'm concerned that referencing the alias like this directly from `node_modules` may cause some side effects. It seems a bit unconventional.

Reply:
I also hate it.  Open to suggestions.

Reply:
It's been fairly normal for a while in bundlers. I think it is ok for now.